Jenkinsfile

```diff
@@ -480,7 +480,7 @@ try {
                                     bat "move package\\win32\\build\\${packageName}-RelWithDebInfo.exe package\\win32\\build\\${packageName}.exe"
                                     bat "move package\\win32\\build\\${packageName}-RelWithDebInfo.zip package\\win32\\build\\${packageName}.zip"
 
-                                    // Explicitly assume the IDE alias of the main `build` account
+                                    // Explicitly assume the IDE alias of the main `build` role
```

Comment:
Not quite accurate. This is an actual IAM role named `ide-build`.
The Jenkins credential `aws-build-role`, in the IDE folder, overrides the global credential and sets it to the `ide-build` IAM role.